paket.dependencies

```diff
@@ -48,3 +48,25 @@ group Test
   nuget FSharp.Core redirects: force
 
   github forki/FsUnit FsUnit.fs
+
+  
+group netcore
```

Comment:
I should probably unify this group with the main group again, otherwise the netcore build may target different versions ...